hyperframe/frame.py

```diff
@@ -88,13 +88,16 @@ def __repr__(self):
         )
 
     @staticmethod
-    def parse_frame_header(header):
+    def parse_frame_header(header, strict=False):
         """
         Takes a 9-byte frame header and returns a tuple of the appropriate
         Frame object and the length that needs to be read from the socket.
 
         This populates the flags field, and determines how long the body is.
```

Comment:
Can you add a `versionchanged` stanza in this docstring to indicate what version we changed this and what the change was? In this case, the version we changed it would be `5.0.0`.